connector/src/main/java/org/jboss/as/connector/deployers/ra/RaDeploymentActivator.java

```diff
@@ -55,6 +56,9 @@
  * @author <a href="mailto:stefano.maestri@redhat.com">Stefano Maestri</a>
  */
 public class RaDeploymentActivator {
+
+    public static final int CONFIGURE_JDBC_DRIVER_MANAGER_ADAPTER = 0x0180;
```

Comment:
Remove this. See comment below.

Reply:
+1 in my recent DUP refactorings I have found a number of cases where the IDs used in WildFly don't get passed back to WildFly Core where we say we will come back to it later.

Please get a PR raised against WildFly Core and we should be able to get it merged quickly.